### PR TITLE
Fix configlet lint errors

### PR DESCRIPTION
**Pull request by ryanplusplus**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1128,12 +1128,12 @@
...
     {
       "title": "Fast and lightweight",
-      "content": "Lua is known to be one of the fastest amongst interpreted dynamic languages and its interpreter, built with all standard libraries, takes under 300K.",
+      "content": "Lua is one of the fastest interpreted dynamic languages. Its compiler and interpreter fit in 300K.",
       "icon": ""
     },
     {
       "title": "Simple",
-      "content": "Lua has a simple, yet powerful syntax which completly fits into the head, which makes it easy and fast to learn.",
+      "content": "Lua has a simple, yet powerful syntax that is it easy and learn and master.",
```
Comment (imolein) [inline config.json:1136]: Little typo: `... that is it easy ...` to `... that is easy ...`

**Review by imolein — CHANGES_REQUESTED** [review]

File: config.json (hunk 1 above)
Comment (ryanplusplus) [reply config.json:1136]: Whoops, thanks for the catch!

Comment (ryanplusplus) [reply config.json:1136]:
```suggestion
      "content": "Lua has a simple, yet powerful syntax that is it easy to learn and master.",
```

File: config.json [hunk 2]
```diff
@@ -1128,12 +1128,12 @@
...
     {
       "title": "Fast and lightweight",
-      "content": "Lua is known to be one of the fastest amongst interpreted dynamic languages and its interpreter, built with all standard libraries, takes under 300K.",
+      "content": "Lua is one of the fastest interpreted dynamic languages. Its compiler and interpreter fit in 300K.",
       "icon": ""
     },
     {
       "title": "Simple",
-      "content": "Lua has a simple, yet powerful syntax which completly fits into the head, which makes it easy and fast to learn.",
+      "content": "Lua has a simple, yet powerful syntax that is it easy to learn and master.",
```
Comment (imolein) [inline config.json:1136]: But the `it` is still there. Or is this correct? (I'm no native English speaker) :)

Comment (ryanplusplus) [reply config.json:1136]: Whoops, I'm just a dunce.

Comment (ryanplusplus) [reply config.json:1136]:
```suggestion
      "content": "Lua has a simple, yet powerful syntax that is easy to learn and master.",
```

**Review by imolein — APPROVED** [review]
lgtm :)

**Pull request merged**